### ISSUE TITLE
README.md clarification on the attributes

### ISSUE THREAD

**Issue by lorenmh** [mingchen/django-cas-ng#109]
I was wondering how to access additional attributes provided by the CAS server, looked through the documentation but didn't find anything referring to accessing the attributes.

Looked at the source code for the CASBackend and it looks like [there is a session field 'attributes' which is set on authentication](https://github.com/mingchen/django-cas-ng/blob/master/django_cas_ng/backends.py#L24). 

You might want to say this in README.md.

Thank you for writing the library by the way!

**Comment by Elarnon** [comment]
Hm, it looks like the attributes are actually set *before* authentication takes place (i.e. even if the authentication fails for a later reason), which is probably not a good practice.

Anyways, an alternative way to access the attributes is to use the [`cas_user_authenticated`](https://github.com/mingchen/django-cas-ng/blob/70dce32/django_cas_ng/backends.py#L56) signal, which gets passed (amongst other things) the attributes (it is defined in [`django_cas_ng.signals`](https://github.com/mingchen/django-cas-ng/blob/70dce32dd5d816537cbe4d233ec52561c7088de4/django_cas_ng/signals.py)).

**Comment by noah-de** [comment]
I was looking at getting additional attributes as well, I took a look at the [cas_user_authenticated ](https://github.com/mingchen/django-cas-ng/blob/70dce32/django_cas_ng/backends.py#L56) signal and there isn't much in the returned attributes.

It appears as though the user has been authenticated, and has not performed the 2nd trip to the CAS server to present the ticket (which it was issued by the CAS server). What I am referring to is the GET request that returns the XML document (shown in this web flow diagram from [apereo](https://apereo.github.io/cas/5.0.x/protocol/CAS-Protocol.html#web-flow-diagram)).

I have not found a place in this library that performs that 2nd trip to the server to present the ticket. Is this a correct conclusion?

If so, I would like to help develop that functionality.

**Comment by michaelsteigman** [comment]
@Elarnon or @noah-de - have you made any further progress on this (documentation or otherwise)? I'm able to authenticate users but newly created users have no attributes other than username set by default (is this expected?).

I also was looking at the signal `cas_user_authenticated` as a possible solution but the receiver I've configured is not getting called. I've yet to subclass CASBackend to troubleshoot but I saw your posts and thought I'd check and see if you've made any progress.
